app/views/users/new.html.haml

```diff
@@ -1,14 +1,43 @@
+- content_for :head do
+  %script(type="text/javascript" src="https://js.stripe.com/v2/")
+  %script(type="text/javascript")
+    Stripe.setPublishableKey("#{ENV['STRIPE_PUBLISHABLE_KEY']}");
+
+= javascript_include_tag 'payments'
+
 %section.register.container
-  .row
-    .col-sm-8.col-sm-offset-1
-      = bootstrap_form_for @user, layout: :horizontal do |f|
-        %header
-          %h1 Register
-        %fieldset
+  = bootstrap_form_for @user, html: {class: 'form-horizontal', id: 'payment-form'} do |f|
```

Comment:
`layout: :horizontal` was a good choice before. But this way it works too, except layout: :horizontal sets some defaults for you.